### ISSUE TITLE
draggable numbers: keyboard accessibility and keyboard entry

### ISSUE THREAD

**Issue by declann** [declann/ActuarialPlayground.com#1]
- You cannot tab to the draggable numbers
- Keyboard entry would make them suitable for more intentional as well as exploratory changes

The code I use for draggable numbers is modified from Amit Patel - his blog is informative, especially: https://www.redblobgames.com/making-of/draggable/

There is other active work on similar widgets, e.g. [for marimo](https://www.linkedin.com/posts/vincentwarmerdam_got-a-fun-pr-in-this-morning-and-with-that-activity-7303503385704517633-qdL6?utm_source=share&utm_medium=member_desktop&rcm=ACoAAAstp_QBJcVCzB0sguky_6JeFCMH5wU23UM)
